conda/environments/cuml_dev_cuda10.1.yml

```diff
@@ -25,8 +25,9 @@ dependencies:
 - dask-cudf=0.16*
 - dask-ml
 - ucx-py=0.16*
+- ucx-proc=*=gpu
 - nccl>=2.5
-- libcumlprims=0.16.0a200720
```

Comment:
Shouldn't this stay as 0.16?

Reply:
oops I totally need to transition my brain to put 0.16 in places now